packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/index.md

```diff
@@ -4,3 +4,5 @@ layout: Guides
 sections:
  - main
 ---
+
+<!--File needs review-->
```

Comment:
Where is all of the content to be reviewed? Is the Netlify link the only way to review the content? Thanks!

Additionally - our heading style standard is to not number headings. what is the reasoning behind this? Thank you!

Reply:
Yes that's right Susan. I'll sync up with you to discuss other review options.

Reply:
Numbers removed.